bugbug/db.py

```diff
@@ -80,3 +84,32 @@ def delete(path, match):
 
     os.unlink(path)
     os.rename(f'{path}_new', path)
+
+
+def is_outdated(path):
+    ver_path = 'data/DB_VERSION.json'
+
+    if not os.path.exists(ver_path):
+        return True
+    else:
+        with open(ver_path) as db:
+            ver = json.load(db)
+        if ver.get(path):
+            return not DATABASES[path]['version'] == ver[path]
+        else:
+            return True
+
+
+def update_ver_file(ver_path, db_path):
+    if not os.path.exists(ver_path):
```

Comment:
At the end of this branch, you should return, as it's unneeded to reread and rewrite the file.